### ISSUE TITLE
Scroll not properly after some text

### ISSUE THREAD

**Issue by bhaveshrathod27691** [KennethTsang/GrowingTextView#35]
![img_2887](https://user-images.githubusercontent.com/28642470/42641558-6be2f7dc-8612-11e8-887e-bcd7f7d09eaf.PNG)

My TextView is Feedback/comment  enter some text after scrollview not seen Properly I think so after maxHeight it scroll automatically.

**Comment by KennethTsang** [comment]
Did you set you top and bottom constraints properly?

**Comment by bhaveshrathod27691** [comment]
yes but scroll content some lines but after text overlap on top objects and
bottom objects .
